eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -202,6 +202,8 @@ begin {
         "azure-genomics",
         "azure-hdinsight",
         "azure-hdinsight-rserver",
+        "azure-health-insights-cancerprofiling",
```

Comment:
Those doc slugs aren't currently in the list supported by the docs team at https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main so adding them here isn't going to help your docs publishing process.

Reply:
It does look like azure-health-insights is supported and prehaps you should simply add and use that one.

Reply:
thanks
I updated the product slug to azure-health-insights.